internal/characters/cyno/attack.go

```diff
@@ -0,0 +1,207 @@
+	switch attackIndex {
+	case 0:
+		return combat.NewCircleHit(c.Core.Combat.Player(), 1.8, false, combat.TargettableEnemy, combat.TargettableGadget)
+	case 1:
+		return combat.NewCircleHit(
+			c.Core.Combat.Player(),
+			1.35,
+			false,
+			combat.TargettableEnemy,
+			combat.TargettableGadget,
+		) // supposed to be box x=1.8,z=2.7
+	case 2:
+		return combat.NewCircleHit(
+			c.Core.Combat.Player(),
+			1.8,
+			false,
+			combat.TargettableEnemy,
+			combat.TargettableGadget,
+		) // both hits supposed to be box x=2.2,z=3.6
+	case 3:
+		return combat.NewCircleHit(c.Core.Combat.Player(), 2.3, false, combat.TargettableEnemy, combat.TargettableGadget)
+	}
+	panic("unreachable code")
+}
+
+const burstHitNum = 5
+
+var (
+	attackBFrames          [][]int
+	attackBHitmarks        = [][]int{{12}, {14}, {18}, {5, 14}, {40}}
+	attackBHitlagHaltFrame = [][]float64{{0.01}, {0.01}, {0.03}, {0.01, 0.03}, {0.05}}
+	attackBDefHalt         = [][]bool{{false}, {false}, {false}, {false, false}, {true}}
+	attackBStrikeType      = [][]combat.StrikeType{
+		{combat.StrikeTypeSlash},
+		{combat.StrikeTypeSlash},
+		{combat.StrikeTypeBlunt},
+		{combat.StrikeTypeSlash, combat.StrikeTypeSlash},
+		{combat.StrikeTypeBlunt},
+	}
+)
+
+func init() {
+	// NA cancels (burst)
+	attackBFrames = make([][]int, burstHitNum)
+	attackBFrames[0] = frames.InitNormalCancelSlice(attackBHitmarks[0][0], 28) // N1 -> CA
+	attackBFrames[0][action.ActionAttack] = 16
+
+	attackBFrames[1] = frames.InitNormalCancelSlice(attackBHitmarks[1][0], 35) // N2 -> CA
+	attackBFrames[1][action.ActionAttack] = 31
+
+	attackBFrames[2] = frames.InitNormalCancelSlice(attackBHitmarks[2][0], 41) // N3 -> N4
+	attackBFrames[2][action.ActionCharge] = 39
+
+	attackBFrames[3] = frames.InitNormalCancelSlice(attackBHitmarks[3][0], 36) // N4 -> CA
+	attackBFrames[3][action.ActionAttack] = 27
+
+	attackBFrames[4] = frames.InitNormalCancelSlice(attackBHitmarks[4][0], 62) // N5 -> N1
+	attackBFrames[4][action.ActionCharge] = 500                                // illegal action
+}
+
+func (c *char) attackB(p map[string]int) action.ActionInfo {
+	c.tryBurstPPSlide(attackBHitmarks[c.normalBCounter][len(attackBHitmarks[c.normalBCounter])-1])
+
+	for i, mult := range attackB[c.normalBCounter] {
+		ai := combat.AttackInfo{
+			ActorIndex:         c.Index,
+			Abil:               fmt.Sprintf("Pactsworn Pathclearer %v", c.normalBCounter),
+			AttackTag:          combat.AttackTagNormal,
+			ICDTag:             combat.ICDTagNormalAttack,
+			ICDGroup:           combat.ICDGroupDefault,
+			StrikeType:         attackBStrikeType[c.normalBCounter][i],
+			Element:            attributes.Electro,
+			Durability:         25,
+			HitlagFactor:       0.01,
+			HitlagHaltFrames:   attackBHitlagHaltFrame[c.normalBCounter][i],
+			CanBeDefenseHalted: attackBDefHalt[c.normalBCounter][i],
+			Mult:               mult[c.TalentLvlBurst()],
+			FlatDmg:            c.Stat(attributes.EM) * 1.5, // this is A4
```

Comment:
add IgnoreInfusion: true at the very bottom because his infusion cannot be overridden